declarations/WebpackOptions.d.ts

```diff
@@ -1133,6 +1133,10 @@ export interface FileCacheOptions {
 	 * When to store data to the filesystem. (pack: Store data when compiler is idle in a single file).
 	 */
 	store?: "pack";
+	/**
+	 * A function to filter which cache entries should be stored. Return false to skip storing a cache entry.
+	 */
+	storeFilter?: (identifier: string, data: any) => boolean;
```

Comment:
Function signature is similar to the one suggested by @alexander-akait [here](https://github.com/webpack/webpack/issues/19359#issuecomment-2755764498) but I'm not sure if there are any use-cases for filtering on `data`. 

Need input from the maintainers here. If it isn't required, I'll go ahead and remove the `data` parameter.